packer/bsd/netbsd_vanilla.pkr.hcl

```diff
@@ -106,6 +106,14 @@ build {
     inline = ["chmod 744 /etc/rc.local && chmod 744 /etc/rc.shutdown"]
   }
 
+  provisioner "shell" {
+    inline = ["/usr/bin/sed -i 's/PermitRootLogin yes/PermitRootLogin prohibit-password/g' /etc/ssh/sshd_config"]
+  }
+
+  provisioner "shell" {
```

Comment:
Please add a comment explaining what this does, -C isn't that explanatory.

Reply:
Done.